cmd/client.cc

```diff
@@ -116,15 +116,16 @@ int main() {
 
   uint32_t *msg_num = (uint32_t *)&data_buf;
 
+
   while (true) {
-    for (int i = 0; i < 12; i++) {
-      (*msg_num)++;
-      auto data = bytes(data_buf, data_buf + sizeof(data_buf));
+    for (int i = 0; i < 10; i++) {
```

Comment:
This 10 seems magic. I assume it has some relation to the sleep, but it would be nice if this had a name.